### ISSUE TITLE
after 2.3.5 update Cannot read properties of undefined (reading 'Html5QrcodeScanner')

### ISSUE THREAD

**Issue by hermawan-n** [mebjas/html5-qrcode#696]
**Describe the bug**
Before update, my program is running well. Then suddenly some error happen, and program couldn't render "reader" neither read upload file to scan QR.

**To Reproduce**
1. use <script src="https://unpkg.com/html5-qrcode" type="text/javascript"></script>
2. then use simple scanning without ID
3. and error messages will show on console

**Expected behavior**
"reader" class will open camera for QR Code Scanner, and could read decoded text then return decoded result

**Screenshots**
![image](https://user-images.githubusercontent.com/68059443/218248634-d6c24acc-575b-4c2f-a2b5-c89d78ff3691.png)
![image](https://user-images.githubusercontent.com/68059443/218248654-2e0a1ebc-67a9-423c-a59f-98ee036d4eef.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Chrome

**Smartphone (please complete the following information):**
 - Device: iPhone, Android
 - OS: IOS, Android
 - Browser Chrome

**Additional context**
Basically I tried to check with multiple device but still got the same errors

**Comment by DottiestDuke47** [comment]
I've just also run into this issue, was using this library in a kiosk but it's stopped working all of a sudden.

**Comment by DottiestDuke47** [comment]
Temporarily you can use the older release:
https://unpkg.com/html5-qrcode@2.3.4/html5-qrcode.min.js

**Comment by hermawan-n** [comment]
@DottiestDuke47 
Thank you so much for the helps, it works!

**Comment by sergioqb1O1** [comment]
Sam problem after update in different devices. Using older release suggested by @DottiestDuke47 solves the problem

**Comment by bensontrent** [comment]
You can see the problem by comparing the latest release file hosted at:

https://unpkg.com/html5-qrcode

to the previous version hosted at:

https://unpkg.com/html5-qrcode@2.3.4/html5-qrcode.min.js

**Comment by mebjas** [comment]
My bad, wrong thing got published.

Fixed with version 2.3.6

I'll add some checks in place to make sure this doesn't happen again!

Thanks for reporting!

**Comment by mebjas** [comment]
My bad, wrong thing got published.

Fixed with version 2.3.6

I'll add some checks in place to make sure this doesn't happen again!

Thanks for reporting!

(Only published to npm at this point, I am in a flight - will publish to Github release ASAP.)

**Comment by kiran-shivanna** [comment]
we too are facing this issue, scanner opens up but doesn't scan/detect any images, when can we expect the fix to be published to Github release please ?

**Comment by mebjas** [comment]
its published now.

**Comment by bensontrent** [comment]
Thanks @mebjas  You may want to update https://github.com/mebjas/html5-qrcode/blob/master/minified/html5-qrcode.min.js as well.

**Comment by mebjas** [comment]
Will do, thanks!

**Comment by mischro** [comment]
Thank you. Great Project!

**Comment by Tudodesk** [comment]
The QR code scanner loads, the scanner shows on screen but it is no longer detecting the QR code when scanned, no errors if i roll back to previous version it works?

**Comment by mnachfolger** [comment]
I'm having the same issue, but when I try using a previous version, it's still not working now -even though it was working fine with no issues before.

**Comment by mnachfolger** [comment]
I'm having the same issue, but when I try using a previous version, it's still not working now -even though it was working fine with no issues before.

**Comment by MN228** [comment]
I'm having the same issue - it loads but it doesn't scan. When I try using a previous version, it's still not working though-even though it was working fine with no issues before.

In my project I'm using:

<script src="https://raw.githubusercontent.com/mebjas/html5-qrcode/master/minified/html5-qrcode.min.js"></script>
<script src="https://unpkg.com/html5-qrcode"></script>

when I changed the first one to the previous version
<script src="https://raw.githubusercontent.com/mebjas/html5-qrcode/v2.3.4/minified/html5-qrcode.min.js"></script>
it's still not working.

Please help, it's really causing us issues.

Thanks

**Comment by mebjas** [comment]
This should be fixed now. Closing.

Once again, sorry for the inconvenience.

As this project has now become bigger and has more users - I need to put in more quality control gates in place instead of pushing changes from airport.
